book/Ghostfire Gaming; Grim Hollow Player's Guide.json

```diff
@@ -20822,7 +20822,7 @@
 				{
 					"name": "Unarmed Strike",
 					"entries": [
-						"{@atk mw} {@hitYourSpellAttack} to hit, reach 5 ft., one target in the swarm's space. {@h}8 ({@damage 2d6 + 1}) damage."
+						"{@atk mw} {@hitYourSpellAttack} to hit, reach 0 ft., one target in the swarm's space. {@h}8 ({@damage 2d6 + 1}) damage."
```

Comment:
Swarms very typically have ranges of 0ft as they also have the `Swarm` trait which allows *and requires* them to occupy another creature's space to attack, this has a significant mechanical affect regarding attacks of oppourtunity.
For examples see the various swarm statblocks in the 5e SRD from page 389 onwards. https://media.wizards.com/2023/downloads/dnd/SRD_CC_v5.1.pdf

The point; are these changes being made on the basis of checking published (and up to date/errata-ed) material, or are they personal revisions?